### DISCUSSION TITLE
XPath filter to match SVG element

### DISCUSSION THREAD

**Discussion by cweiske** [uBlockOrigin/uBlock-issues#2077] [Q&A] [answered]
https://www.heise.de/ lists articles on its main page, and among them "heise+" articles that are only available to subscribers. I'd like to hide them.

They can be recognized by the "heise+" icon, but that icon is an embedded svg file.

My XPath does not recognize the SVG tag, presumably because it's in a different namespace:
```
##:xpath(//article[.//svg])
```

Does [ublock's xpath](https://github.com/gorhill/uBlock/wiki/Procedural-cosmetic-filters#subjectxpatharg) support namespaces? How can I use them?

**Comment by mapx-** [comment, marked as answer]
You could use:

`heise.de##[viewBox*="194"]:upward(article)`

**Reply by cweiske** [reply]
Thank you.
